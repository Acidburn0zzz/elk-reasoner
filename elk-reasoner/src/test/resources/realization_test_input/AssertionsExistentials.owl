Prefix(:=<http://example.org/>)
Ontology(<www.example.org#assertions_existentials>
ObjectPropertyAssertion(:R :a :b)
ObjectPropertyAssertion(:S :b :a)
ClassAssertion(:A :a)
ClassAssertion(:B :b)
SubClassOf(ObjectSomeValuesFrom(:R :B) :C)
SubClassOf(ObjectSomeValuesFrom(:S :C) :D)
)
