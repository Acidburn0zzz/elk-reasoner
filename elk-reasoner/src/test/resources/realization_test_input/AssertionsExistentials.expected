Prefix(:=<http://example.org/>)
Ontology(<www.example.org#assertions_existentials#taxonomy>
ClassAssertion(:A :a)
ClassAssertion(:B :b)
ClassAssertion(:C :a)
ClassAssertion(:D :b)
)